libs/domain/cart/entry/src/entry.component.ts

```diff
@@ -105,7 +106,7 @@ export class CartEntryComponent
     if (!this.$options()?.enableItemImage) return;
 
     return html`
-      <a href=${this.$productLink()}>
+      <a href=${ifDefined(this.$productLink())}>
```

Comment:
```suggestion
      <a .href=${this.$productLink()}>
```

This seems to work fine, where the `ifDefined` comes for free. I'm not sure how this works, if there are some properties that will be reflected as attributes.

Reply:
I hope we want to keep at as attribute not as property in order of ssr. But the main point of ifDefined it's that attribute will be not rendered if value is null or undefined

Reply:
But I can drop ofc